### ISSUE TITLE
IllegalFormatConversionException with LocalProteinDomainParser

### ISSUE THREAD

**Issue by yamule** [biojava/biojava#1009]
I met IllegalFormatConversionException when I tried to use LocalProteinDomainParser in biojava-structure-6.0.3.jar .
```
java.util.IllegalFormatConversionException: d != java.lang.Double
	at java.util.Formatter$FormatSpecifier.failConversion(Formatter.java:4302)
	at java.util.Formatter$FormatSpecifier.printInteger(Formatter.java:2793)
	at java.util.Formatter$FormatSpecifier.print(Formatter.java:2747)
	at java.util.Formatter.format(Formatter.java:2520)
	at java.io.PrintStream.format(PrintStream.java:970)
	at java.io.PrintStream.printf(PrintStream.java:871)
	at org.biojava.nbio.structure.domain.pdp.ClusterDomains.cluster(ClusterDomains.java:189)
	at org.biojava.nbio.structure.domain.LocalProteinDomainParser.suggestDomains(LocalProteinDomainParser.java:104)
	at org.biojava.nbio.structure.domain.LocalProteinDomainParser.suggestDomains(LocalProteinDomainParser.java:69)
	at com.mycompany.bijavacheck.MyMainClass.main(MyMainClass.java:29)
```
Possibly, the line https://github.com/biojava/biojava/blob/master/biojava-structure/src/main/java/org/biojava/nbio/structure/domain/pdp/ClusterDomains.java#L189
`				if(verbose) System.out.printf(" Maximum value is less than cut off value. (max:%d)%n", maximum_value);`
should be
`				if(verbose) System.out.printf(" Maximum value is less than cut off value. (max:%f)%n", maximum_value);`
.
(It looks fine in dev branch's one but its commit is older than that of main so I'm reporting this.)

**Comment by aalhossary** [comment]
OK I'll handle it.

**Comment by heuermh** [comment]
Fixed by #1010, thank you @yamule @aalhossary
